cypress/features/regression/stepSequence.feature

```diff
@@ -3,7 +3,7 @@ Feature: Step Sequence component
 
   @positive
   Scenario Outline: I set orientation to <orientation>
-    When I open default "Step Sequence" component in noIFrame with "stepSequence" json from "commonComponents" using "<nameOfObject>" object name
+    When I open step-sequence "Step Sequence Test" component in noIFrame with "stepSequence" json from "commonComponents" using "<nameOfObject>" object name
```

Comment:
this file needs to be removed as we have covered these tests by `chromatic`, also needs to be removed used objects from the  `cypress/fixtures/commonComponents/stepSequence.json`